### PR TITLE
Added basic pytests and circleci

### PR DESCRIPTION
**Pull request by psavery**
There is now a test for creating and getting molecules,
and one for creating and getting calculations.

They pass in my usage of them.

### CODE REVIEW

**Comment by psavery** [comment]
Let me know what you think.

**Comment by psavery** [comment]
@cjh1 I updated the tests. Let me know what you think.

**Comment by psavery** [comment]
I just pushed some more changes. It's almost there, but still doesn't quite work. Here is the current state of my tests:

https://circleci.com/gh/psavery/mongochemserver/9

File: devops/docker/Dockerfile-circleci [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# We need to install girder_pytest...
+RUN git clone https://github.com/girder/girder.git /girder_local && \
+  cd /girder_local && \
+  pip install --no-cache-dir -r requirements-dev.txt
+
+# Get mongodb
+RUN apt-get update && apt-get -y install mongodb-server
+
+# Install clean up
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```
Comment (cjh1) [inline devops/docker/Dockerfile-circleci]: The convention here would be to name it Dockerfile.circleci

File: girder/molecules/plugin_tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+# Our method for creating a molecule
+@pytest.fixture
+def molecule(user):
+    from girder.plugins.molecules.models.molecule import Molecule
+    from girder.plugins.molecules import openbabel
+
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+
+    with open(dir_path + '/data/ethane.xyz', 'r') as rf:
+        xyzData = rf.read()
```
Comment (cjh1) [inline girder/molecules/plugin_tests/__init__.py]: snake_case

File: girder/molecules/plugin_tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    input_format = 'xyz'
+    data = xyzData
```
Comment (cjh1) [inline girder/molecules/plugin_tests/__init__.py]: here as well

**Comment by psavery** [comment]
Closed in favor of PR #53.

**Pull request closed without merging**